docs/configuration/memory.md

```diff
@@ -154,6 +184,15 @@ View memory usage by using the `--debug` flag :)
 
 
 ## 🧠 Memory pre-seeding
+
+!!! warning
+    Data ingestion is broken in v0.4.5 and possibly earlier versions. This is a known issue that will be addressed in future releases. PRs welcome.
+    [Issue 4435](https://github.com/Significant-Gravitas/Auto-GPT/issues/4435)
```

Comment:
I don't think we want PRs for this quite yet.

Reply:
I consulted with Luke on this

Reply:
@lc0rp I don't think we're ready for data ingestion.  @Pwuts is now working on integrating memory back into the system and we should obviate the need for the data ingestion script as part of that work (I would think @Pwuts should confirm).

Reply:
Data ingestion doesn't have any value as long as memory isn't being used. Same goes for PRs that fix data ingestion. It's a black hole at the moment. I'd advise to remove the data ingestion script and add a corresponding warning/notice in the bulletin and docs.

Reply:
@Pwuts So how should I proceed with this? Remove the data ingestion portion entirely and add a notice saying it has been removed?

Reply:
This all makes sense.

@NeonN3mesis, it's still useful to let people know it's broken. I'll suggest a change.